hasjob/templates/reject_domain_email.html

```diff
@@ -0,0 +1,15 @@
+{% extends "inc/email_layout_lite.html" %}
+
+{% block content %}
+  <p>Hi {{ post.company_name }},</p>
```

Comment:
Make this `Hi {% if post.user %}{{ post.user.fullname }}{% else %}{{ post.company_name }}{% endif %}`,